source/extensions/config_subscription/grpc/grpc_mux_context.h

```diff
@@ -31,6 +31,7 @@ struct GrpcMuxContext {
   BackOffStrategyPtr backoff_strategy_;
   const std::string& target_xds_authority_;
   EdsResourcesCachePtr eds_resources_cache_;
+  bool skip_subsequent_node_;
```

Comment:
Can we split this PR to 2:
1) refactor the `skip_subsequent_node_` here.
2) implement the fix for delta-xDS.